src/Symfony/Component/VarDumper/Cloner/Data.php

```diff
@@ -210,7 +209,6 @@ private function dumpItem($dumper, $cursor, &$refs, $item)
      *
      * @param DumperInterface $dumper
      * @param Cursor          $parentCursor The cursor of the parent hash
-     * @param array           &$refs        A map of all references discovered while dumping
```

Comment:
@TomasVotruba Checking the diff, I think the current strategy is too aggressive. Here is one such example where the comment seems interesting and should probably be kept. WDYT?

Reply:
@fabpot Thanks for comment. I've noticed this already. I think it's caused by `&` which probably leads to empty comment.

I'm adding to the cases to be fixed